web_widget_x2many_2d_matrix/__manifest__.py

```diff
@@ -4,7 +4,7 @@
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
 {
     "name": "2D matrix for x2many fields",
-    "version": "11.0.1.1.2",
+    "version": "11.0.1.1.3",
```

Comment:
this is a feature, not a patch. Please revert this change: we can handle proper number by merging w/ OCA bot ;)